rtloader/common/stringutils.c

```diff
@@ -14,6 +14,15 @@ PyObject * ydump = NULL;
 PyObject * loader = NULL;
 PyObject * dumper = NULL;
 
+/**
+ * returns a C (NULL terminated UTF-8) string from a python string.
+ * 
+ * @param object  A Python string to be converted to C-string.
```

Comment:
nit: we seem to use the `\` instead of `@` in other files as the doxygen command prefix, I think we should try to be consistent across files.